RELEASE-NOTES.md

```diff
@@ -1,3 +1,59 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
+  [riak_cs/#1041](https://github.com/basho/riak_cs/pull/1041). This
+  change prevents unnecessary siblings increase when backpressure is
+  triggered under high upload concurrency, and uploads are interleaved
+  during the backpressure sleep.
```

Comment:
How do you think about adding that the bug only affect concurrent normal PUT Object but does not do multipart uploads?